less/mixins.less

```diff
@@ -1 +1,25 @@
-// Mixins Here
\ No newline at end of file
+// Mixins Here
+.btn-mxn(@size, @wide) {
+    color: white;
+    background-color: @btn-bg-color;
+    padding: @size;
+    width: @wide;
+    border-radius: 5px;
+    font-size: 1.7rem;
+    border: 1px solid @btn-bg-color;
+    text-align: center;
+}
+
+.btn-hvr() {
```

Comment:
Great use of parametric mixins here. You actually could move this into the `.btn-mixin`

Mixins give you full access to the whole power of Less. So you actually could move the `:hover` psuedo selector, and, breakpoints into the mixin.

This way all you really needed for the button would be to apply your mixin, and pass it the right arguments to create the custom styling on that button.